mkosi.build

```diff
@@ -56,6 +56,21 @@ for bpftool in /usr/lib/linux-tools/*/bpftool; do
         break
 done
 
+# CentOS Stream 8 includes bpftool 4.18.0 which is lower than what we need. However, they've backported the
```

Comment:
I think it's fine to keep this in mkosi.build. Anyone backporting stuff from bpftool can easily patch systemd's version requirement, in fact that's exactly what we do in the centos hyperscale sig.

If bpftool had a proper introspection command we could just that for feature detection but without that we have to start parsing it's CLI output which is just ugly so I'd like to avoid having to do that.

Reply:
I see. That makes sense to me.